File: backend/tildau/src/main/java/kz/kbtu/tildau/dto/course/ExerciseResponse.java
package kz.kbtu.tildau.dto.course;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Getter
public class ExerciseResponse {
    private UUID id;
    private String title;
    private String instruction;
}

File: android/Tildau2/app/src/main/java/com/example/tildau/data/model/course/ExerciseResponse.kt
package com.example.tildau.data.model.course

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ExerciseResponse(
    val id: String,
    val title: String,
    val instruction: String,
    @SerializedName("completed")
    val isCompleted: Boolean,

    @SerializedName("locked")
    val isLocked: Boolean
) : Serializable
